src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setBall(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
```

Comment:
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```

why setBall? Setters used for some one field, or if you want to create and initialize Ball instance with several fields in a time, use constructor